app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -158,6 +192,24 @@ class BookmarksTest {
         }
     }
 
+    @Test
+    fun threeDotMenuShareBookmarkTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        browserScreen {
+            createBookmark(defaultWebPage.url)
+        }.openThreeDotMenu {
+        }.openLibrary {
+        }.openBookmarks {
```

Comment:
Is an idling resource needed here too, because we're waiting for the populated list (even thought it's one bookmark)?